src/theme/colors.ts

```diff
@@ -39,12 +39,9 @@ const colors = {
       success: "#00D1A2" /* Unused */,
       attention: "#D6C71C",
       error: "#FF3C53",
-      select: "#0EB28E" /* Untested/Unused */,
+      select: "#3B3CD0" /* Untested/Unused */,
```

Comment:
Sorry, for any colors here that you KNOW you have used can you remove the comments saying it is unused or untested in this file?
I put them there when I updated these colors to match Figma and I noticed at that point not all the colors were being used yet.

Reply:
@KaWaite not all the colors are used for sure, still one and two color maybe not used. But now the colors here are the same with the design, so which is good I think. I also change some color based on our service. I will remove the commet s here